lib/3scale/backend/logging/external/impl/bugsnag.rb

```diff
@@ -56,7 +54,7 @@ def configure(api_key)
                   config.timeout = 3
                   config.logger = Backend.logger
                   config.meta_data_filters = []
-                  config.notify_release_stages = ['production', 'preview']
+                  config.enabled_release_stages = %w[production staging development]
```

Comment:
actually we should get rid of `staging` environment and use `production` in staging. Like we do with porta. Otherwise we risk running a different thing in production and in staging. This separatio of environments only made sense in the old legacy deployment based on I forgot what it already was and don't want to remember.

Reply:
If ops starts using `production` for staging, then we could remove all mentions to staging along the project.

Reply:
we need to request that I guess